packages/yourloops/services/http.ts

```diff
@@ -0,0 +1,51 @@
+import axios, { AxiosRequestConfig, AxiosResponse, AxiosError } from "axios";
+import httpStatus from "../lib/http-status-codes";
+import { t } from "../lib/language";
+
+export default class HttpService {
+  static async get<T>(url: string, config?: AxiosRequestConfig): Promise<AxiosResponse<T>> {
+    try {
+      return axios.get<T>(url, { ...config });
+    } catch (error: any) {
+      throw this.handleError(error);
```

Comment:
```suggestion
      throw HttpService.handleError(error);
```
it's a static method, not a class instance